torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -123,10 +123,16 @@ std::unique_ptr<RpcCommandBase> RequestCallbackImpl::processRpc(
 
       // Attach the 'send' autograd function if needed.
       if (autogradContext != nullptr) {
+        const int kAutoIncrementBits = 48;
+        rpc::worker_id_t dstWorkerId =
+            autogradMetadata.autogradContextId >> kAutoIncrementBits;
         addSendRpcBackward(
```

Comment:
@pritamdamania87 per your comment on the previous version of this diff, this is the callsite where I couldn't quite figure out how to pass in the destination worker. Right now, I am reading it off the autogradMetadata and using the assumption that it is the first 16 bits of the contextId. However, I'm not sure if we want to do it this way, or if there's a better way to do it.